presto-jdbc/src/main/java/com/facebook/presto/jdbc/PrestoConnection.java

```diff
@@ -217,15 +237,17 @@ public void setTransactionIsolation(int level)
             throws SQLException
     {
         checkOpen();
-        throw new SQLFeatureNotSupportedException("Transactions are not yet supported");
+        getIsolationLevel(level);
+        isolationLevel.set(level);
     }
 
+    @SuppressWarnings("MagicConstant")
```

Comment:
I don't see a constant in the method

Reply:
This is a warning that the method is not returning one of the magic constants such as `Connection.TRANSACTION_NONE`